src/ILLink.RoslynAnalyzer/RequiresAnalyzerBase.cs

```diff
@@ -355,6 +361,18 @@ protected bool IsMemberInRequiresScope (ISymbol containingSymbol)
 			return false;
 		}
 
+		/// <summary>
+		/// True if member of a call is considered to be annotated with the Requires... attribute. Primarily for property accessor declarations.
+		/// </summary>
+		/// <param name="containingSymbol">
+		///	Symbol that is either an overriding member or an overriden/virtual member
+		/// </param>
+		protected bool IsOverrideMemberInRequiresScope (ISymbol containingSymbol)
+		{
+			return containingSymbol.HasAttribute (RequiresAttributeName)
+					|| containingSymbol.ContainingType.HasAttribute (RequiresAttributeName);
```

Comment:
Does this need a null check similar to the one added in https://github.com/dotnet/linker/pull/2400? /cc @tlakollo

Reply:
This needs the fix from #2400, it should be on main already